### DISCUSSION TITLE
Virus and threat protection warning

### DISCUSSION THREAD

**Discussion by markoni25** [EDCD/EDMarketConnector#1386] [General]
Hi there

Today I received warning from Windows Security: 
![image](https://user-images.githubusercontent.com/67276144/147743007-23a85123-96e2-41e9-942e-7c1c561396d2.png)
![image](https://user-images.githubusercontent.com/67276144/147743034-bcc06f25-2981-4ca9-bfa4-6b89e59648b6.png)

**Comment by Athanasius** [comment]
https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting#installer-and-or-executables-flagged-as-malicious-viruses
